week0003/thainguyentran/CSharp/week0003.cs

```diff
@@ -6,27 +6,28 @@ namespace DVRK
 {
     class Program
     {
-        public static Dictionary<string, double> clients = new Dictionary<string, double>();
+        public static Dictionary<string, double> clients = new Dictionary<string, double>(StringComparer.OrdinalIgnoreCase);
```

Comment:
Alternatively, you can lower-case/upper-case the string before passing into the dictionary as well :D The choice is yours.